src/libprojectM/ProjectM.cpp

```diff
@@ -541,7 +541,11 @@ int ProjectM::initPresetTools(int gx, int gy)
     /* Set the seed to the current time in seconds */
     srand(time(NULL));
 
-    std::string url = (m_flags & FLAG_DISABLE_PLAYLIST_LOAD) ? std::string() : settings().presetURL;
+    std::string url;
+    if ((m_flags & Flags::DisablePlaylistLoad) == Flags::DisablePlaylistLoad)
```

Comment:
Could just use `!= 0`.

Reply:
Or just `!` after changing it to a boolean.

Reply:
Will fix that at a later point, as it also affects the external API. Will for now try to keep the changes closely to ProjectM.hpp/.cpp.